go/cmd/mysqlctl/mysqlctl.go

```
@@ -42,7 +42,7 @@ var (
 	tabletUID   = flag.Uint("tablet_uid", 41983, "tablet uid")
 	mysqlSocket = flag.String("mysql_socket", "", "path to the mysql socket")
 
-	tabletAddr string
+	tabletAddr string //nolint
```

Is it possible to add a reason? so that we know which error is being ignored?